bazel-mode.el

```diff
@@ -21,6 +21,43 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode-buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
+  :group 'bazel-mode
+  :link '(url-link
+          "https://github.com/bazelbuild/buildtools/tree/master/buildifier"))
+
+(defun bazel-mode-buildifier ()
+  "Format current buffer using buildifier."
+  (interactive "*")
+  (let ((input-buffer (current-buffer))
+        (buildifier-buffer (get-buffer-create "*buildifier*"))
+        ;; Run buildifier on a file to support remote BUILD files.
+        (temp-file (make-nearby-temp-file "buildifier")))
+    (write-region nil nil temp-file nil 1)
+    (with-current-buffer buildifier-buffer
+      (setq-local inhibit-read-only t)
+      (erase-buffer)
+      (unwind-protect
```

Comment:
Move this `unwind-protect` to directly after creating the file so that it's deleted if something were to signal an error in between.

Reply:
Done.